cudamapper/include/claragenomics/cudamapper/index.hpp

```diff
@@ -72,6 +72,16 @@ class Index
     /// \return number of reads in input data
     virtual read_id_t number_of_reads() const = 0;
 
+    /// ToDo this is to allow IndexCache to access to read_id_to_read_name_, may need to change if overloading is not desired
```

Comment:
I would suggest naming it `read_ids_to_read_names()`, not just to avoid overloading but because this function returns something that maps multiple read_ids to multiples read_names.

Reply:
done.